### PR TITLE
Misc. fixes

### PR DESCRIPTION
**Pull request by ghedo**

### CODE REVIEW

**Comment by ghedo** [comment]
Given the second commit, the first one is probably not needed, but it doesn't hurt either.

File: src/ngx_http_v2_push_module.c [hunk 1]
```diff
@@ -723,7 +725,54 @@ ngx_http_v2_push_populate_path(ngx_http_request_t *r, u_char *u_str, size_t u_le
         return NGX_ERROR;
     }
 
-    return ngx_http_process_request_uri(r);
+    if (r->args_start) {
```
Comment (StephanDollberg) [inline src/ngx_http_v2_push_module.c:728]: this is basically just copied from nginx with the `ngx_http_close_request` and `ngx_http_finalize_request` removed?

File: src/ngx_http_v2_push_module.c [hunk 2]
```diff
@@ -723,7 +725,54 @@ ngx_http_v2_push_populate_path(ngx_http_request_t *r, u_char *u_str, size_t u_le
...
+        if (r->uri.data == NULL) {
+            return NGX_ERROR;
+        }
+
+        cscf = ngx_http_get_module_srv_conf(r, ngx_http_core_module);
+
+        if (ngx_http_parse_complex_uri(r, cscf->merge_slashes) != NGX_OK) {
+            r->uri.len = 0;
+
+            ngx_log_error(NGX_LOG_INFO, r->connection->log, 0,
```
Comment (StephanDollberg) [inline src/ngx_http_v2_push_module.c]: guess we can remove this logging as well (shrug)

**Review by StephanDollberg — APPROVED** [review]

File: src/ngx_http_v2_push_module.c (hunk 1 above)
Comment (ghedo) [reply src/ngx_http_v2_push_module.c:728]: Yes.

File: src/ngx_http_v2_push_module.c (hunk 2 above)
Comment (ghedo) [reply src/ngx_http_v2_push_module.c]: Yeah makes sense, the error message is wrong anyway.

**Pull request merged**